Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -1188,28 +1150,19 @@ private static void AssertFileIsNotReferenced(string fullFilePath, string conten
             content.Should().NotContain(formattedPath, "File should not be referenced: {0}", formattedPath);
         }
 
-        #endregion Assertions
-
-        #region Private methods
-
         private string ComputeProjectBaseDir(string teamBuildValue, string userValue, string[] projectPaths)
         {
             var config = new AnalysisConfig();
             var logger = new TestLogger();
             new PropertiesWriter(config, logger);
             config.SonarOutputDir = TestSonarqubeOutputDir;
-
             config.SourcesDirectory = teamBuildValue;
-
-            if (config.LocalSettings == null)
-            {
-                config.LocalSettings = new AnalysisProperties();
-            }
+            config.LocalSettings ??= new();
```

Comment:
🤩

Reply:
That makes it clear why I started with the C# 10 bump 😆